pkg/controller/virtualmachineimport/virtualmachineimport_controller.go

```diff
@@ -255,6 +269,9 @@ func (r *ReconcileVirtualMachineImport) importDisks(provider provider.Provider,
 					if err := r.afterSuccess(vmName, vmiName, provider); err != nil {
 						return err
 					}
+
+					// Emit event vm is successfully imported
+					r.recorder.Event(instance, corev1.EventTypeNormal, EventImportSucceeded, "Virtual Machine import successful")
```

Comment:
it would be good to add vm name so the user would know which vm was imported.
I would rephrase it to "Virtual Machine %s was imported successfully"

Reply:
pls use types.NamespacedName for VM